packages/@sfx/search-driver-plugin/src/search-driver-plugin.ts

```diff
@@ -71,7 +78,7 @@ export default class SearchDriverPlugin implements Plugin {
    * @param event the event whose payload is the search term.
    */
   fetchSearchData(event: CustomEvent<SearchRequestPayload>): void {
-    const searchTerm = event.detail;
+    const searchTerm = event.detail.value;
```

Comment:
Shouldn't this also extract any config object from the view layer and send it to `sendSearchApiRequest()` like the `sayt-driver` does? Or I guess is that out of scope for this ticket?  Is there a ticket planned for this? Also, given that the functions will be mostly the same, should they be stored somewhere shared?

Reply:
This will be added in an upcoming ticket.